python/runtime/model/model.py

```diff
@@ -126,8 +204,8 @@ def _dump_pkl(obj, to_file):
         pickle.dump(obj, f, pickle.HIGHEST_PROTOCOL)
```

Comment:
I'm curious about if `pickle.HIGHEST_PROTOCOL` can work on lower PROTOCOL, like we upload a model created by Py3, and load it by Py2.

Reply:
Great suggestions! Changed to `protocol=2` to be compatible with Py2 and Py3.